gobblin-data-management/src/test/java/org/apache/gobblin/util/commit/SetPermissionCommitStepTest.java

```diff
@@ -55,7 +57,8 @@ public void setUp() throws IOException {
     dir1 = new Path(ROOT_DIR, "dir1");
     this.fs.mkdirs(dir1);
 
-    OwnerAndPermission ownerAndPermission = new OwnerAndPermission("owner", "group", permission);
+    OwnerAndPermission ownerAndPermission = new OwnerAndPermission("owner", "group", permission, null,
+        Lists.newArrayList());
```

Comment:
once more: on more than a few of these... what about the three-arg version you actually retained?

Reply:
latest commit takes care of this!